esmvaltool/recipes/recipe_climwip_test_performance_sigma.yml

```diff
@@ -0,0 +1,190 @@
+  - {project: CMIP6, exp: [historical, ssp585], dataset: GISS-E2-1-G, ensemble: r1i1p1f2, grid: gn}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: GISS-E2-1-G, ensemble: r1i1p5f1, grid: gn}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: HadGEM3-GC31-MM, ensemble: r(1:4)i1p1f3, grid: gn}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: INM-CM4-8, ensemble: r1i1p1f1, grid: gr1}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: INM-CM5-0, ensemble: r1i1p1f1, grid: gr1}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: IPSL-CM6A-LR, ensemble: r(1:4)i1p1f1, grid: gr}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: IPSL-CM6A-LR, ensemble: r6i1p1f1, grid: gr}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: IPSL-CM6A-LR, ensemble: r14i1p1f1, grid: gr}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: KACE-1-0-G, ensemble: r(1:3)i1p1f1, grid: gr}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: MCM-UA-1-0, ensemble: r1i1p1f2, grid: gn}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: MIROC-ES2L, ensemble: r1i1p1f2, grid: gn}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: MIROC6, ensemble: r(1:3)i1p1f1, grid: gn}
+  # DEBUG: fails, unclear why
+  # - {project: CMIP6, exp: [historical, ssp585], dataset: MPI-ESM1-2-HR, ensemble: r(1:2)i1p1f1, grid: gn}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: MPI-ESM1-2-LR, ensemble: r(1:10)i1p1f1, grid: gn}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: MRI-ESM2-0, ensemble: r1i1p1f1, grid: gn}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: MRI-ESM2-0, ensemble: r1i2p1f1, grid: gn}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: NESM3, ensemble: r(1:2)i1p1f1, grid: gn}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: NorESM2-LM, ensemble: r1i1p1f1, grid: gn}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: NorESM2-MM, ensemble: r1i1p1f1, grid: gn}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: TaiESM1, ensemble: r1i1p1f1, grid: gn}
+
+obs_data: &obs_data  # for climwip performance metrics
+  - {dataset: ERA5, project: native6, type: reanaly, version: '1', tier: 3}
+
+preprocessors:
+  climwip_general: &general
+    regrid:
+      target_grid: 2.5x2.5
+      scheme: linear
+    mask_landsea:
+      mask_out: sea
+    extract_region:
+      start_longitude: 0.0
+      end_longitude: 39.0
+      start_latitude: 30.0
+      end_latitude: 76.25
+
+  climatological_mean:
+    <<: *general
+    climate_statistics:
+      operator: mean
+
+  temperature_anomalies:
+    custom_order: true
+    area_statistics:
+      operator: mean
+    annual_statistics:
+      operator: mean
+    anomalies:
+      period: full
+      reference:
+        start_year: 1981
+        start_month: 1
+        start_day: 1
+        end_year: 2010
+        end_month: 12
+        end_day: 31
+      standardize: false
+
+diagnostics:
+  calculate_weights_climwip:
```

Comment:
diagnostics misses description, and provenance tags (themes, realms).
E.g.
```
description: Calculate weights based on performance and independence
themes:
    - phys
realms:
    - atmos
```

Reply:
I pretty much copied that, I have to admit I don't really understand the provenance tags yet